src/olympia/addons/tasks.py

```diff
@@ -577,6 +578,13 @@ def add_static_theme_from_lwt(lwt):
         addon=lwt, activity_log__action__in=rating_activity_log_ids)
     [alog.transfer(addon) for alog in addonlog_qs.iterator()]
 
+    # Copy the ADU statistics - the raw(ish) daily UpdateCounts for stats
+    # dashboard and future update counts, and copy the summary numbers for now.
+    migrate_theme_update_count(lwt, addon)
+    addon_updates.update(
+        average_daily_users=lwt.persona.popularity or 0,
```

Comment:
Hmm, for reference… `persona.popularity` are weekly numbers but both are averages so this should be fine I think.

Reply:
`average_daily_users` is a 2 week average; `persona.popularity` is a 1 week average but they're calculated in a similar way.

https://github.com/mozilla/addons-server/blob/ceae9d4fd0ad3f2cbb8233fd494bd1248fe58854/src/olympia/addons/cron.py#L39:L44

https://github.com/mozilla/addons-server/blob/57915e248080bafdf5b3a3d3f95b78d64bd0c278/src/olympia/stats/management/commands/update_theme_popularity_movers.py#L39:L42

Reply:
Ah, I looked at the wrong file. That works either way.